www_src/pages/make/make.jsx

```diff
@@ -3,23 +3,24 @@ var render = require('../../lib/render.jsx');
 var api = require('../../lib/api.js');
 var Card = require('../../components/card/card.jsx');
 var Loading = require('../../components/loading/loading.jsx');
+var router = require('../../lib/router');
 
 var Make = React.createClass({
-  mixins: [],
+  mixins: [router],
```

Comment:
We've started to do:
```js
mixins: [
    require('../blah/blah')
]
```
in other components as to keep the module namespace a little more terse / clear.